dir.props

```diff
@@ -67,6 +67,9 @@
     <PackagesBinDir>$(__PackagesBinDir)</PackagesBinDir>
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
 
+    <PackageInstallPath>$(__PackageInstallPath)</PackageInstallPath>
```

Comment:
@karajas isn't there a way to disable the installing of this package instead of hacking the path here?